src/status_im/contexts/profile/edit/name/events_test.cljs

```diff
@@ -0,0 +1,15 @@
+(ns status-im.contexts.profile.edit.name.events-test
+  (:require [cljs.test :refer [deftest is]]
+            matcher-combinators.test
+            [status-im.contexts.profile.edit.name.events :as sut]))
```

Comment:
what is `sut`? something more descriptive here would be better imo

Reply:
Just saw it in other events-test files and copied the same, no idea why they used this. any suggestion? @J-Son89

Reply:
ah okay, something simple like `[status-im.contexts.profile.edit.name.events :as .edit-name.events]` is clearer imo

Reply:
@J-Son89, @mohsen-ghafouri, `sut` means System Under Test. It's a convention used in some projects to refer to the file under test. I believe the alias is defined automatically by some tools when Clojure files ending with `_test` are created (this is what happens to me in my Emacs).

As a convention, it's actually quite nice once you get used to it because in unit tests we already know which namespace we're in, so using a long and descriptive alias just makes the code verbose.

Reply:
Sounds good 👍